pkg/server/server.go

```diff
@@ -467,7 +470,7 @@ func (s *Server) backup(backupDirectoryID string, policyID string, name string,
 	// Upload file to server
 	s.reportStartUpload(progressOutput)
 	pw := backupapi.NewProgressWriter(progressOutput)
-	if err := s.backupClient.UploadFile(rp.RecoveryPoint.ID, fi, pw); err != nil {
+	if err := s.backupClient.UploadFile(rp.RecoveryPoint.ID, fi, pw, batch); err != nil {
```

Comment:
I think when having an error (backup and restore), the agent will push a message FAILED to notify that action was failed.

Reply:
Please open another issue for this.